app/js/components/LabOrderListFIlters.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {
+  FormControl,
+} from 'react-bootstrap';
+import CustomDatePicker from './shared/CustomDatePicker';
+import Dropdown from './shared/Dropdown';
+
+class LabOrderListFilters extends PureComponent {
+  renderNameEMROrOrderIdFilter() {
+    const { handleFieldChange, clearNameEMRField, nameField } = this.props;
+    return (
+      <span className="form-filter-group one-third">
+        <span className="filter-label">Search for a sample</span>
```

Comment:
class `filter-label` appears in a number of places but I don't see it in any style file